salt/beacons/btmp.py

```diff
@@ -295,7 +281,7 @@ def beacon(config):
             event = {}
             for ind, field in enumerate(FIELDS):
                 event[field] = pack[ind]
-                if isinstance(event[field], salt.ext.six.string_types):
```

Comment:
Fix for test. Here's an example of wrong usage of `six.string_types` that is expected by this logic to pass for `bytes` as we can see on the next line.

Reply:
I don't know if it would be worthwhile to write the test, but you *could* write a test, probably mocking a bunch of stuff to make it to this point, that would get you some bytes instead of string types/text.

Reply:
The failing test would fail to convert from bytes to unicode/text, of course.